### PR TITLE
migrate amp-addthis sample

### LINKED ISSUE #1777: amp-addthis sample not migrated
https://ampbyexample.com/components/amp-addthis/

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes #1777

### CODE REVIEW

File: examples/source/1.components/amp-addthis.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!--
+  ## Introduction
+  AddThis Floating Bar and Inline Share Buttons are available for AMP! Make it simple for website visitors to share your content with our free social sharing buttons. Allow your visitors to connect directly to 200+ social networks, including Facebook, Twitter, WhatsApp, and more.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="/components/amp-addthis/">
```
Comment (matthiasrohmer) [inline examples/source/1.components/amp-addthis.html]: This can be `<% canonical %>` now 🙂

File: examples/source/1.components/amp-addthis.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    height="92"
+    data-pub-id="ra-5c191331410932ff"
+    data-widget-id="mv93"
+    data-widget-type="inline">
+  </amp-addthis>
+
+  <!-- ## Implementation -->
+  <!--
+  Visit our page to learn to how to setup AddThis on AMP:
+  <https://www.addthis.com/academy/how-to-install-addthis-inline-share-buttons-on-amp-accelerated-mobile-pages/>
```
Comment (matthiasrohmer) [inline examples/source/1.components/amp-addthis.html]: That link syntax (?) looks strange.

File: examples/source/1.components/amp-addthis.html (hunk 1 above)
Comment (sebastianbenz) [reply examples/source/1.components/amp-addthis.html]: sweet! done

File: examples/source/1.components/amp-addthis.html (hunk 2 above)
Comment (sebastianbenz) [reply examples/source/1.components/amp-addthis.html]: fixed it

**Pull request merged**